cmd/neofs-lens/internal/meta/status.go

```diff
@@ -0,0 +1,48 @@
+package meta
+
+import (
+	"fmt"
+
+	common "github.com/nspcc-dev/neofs-node/cmd/neofs-lens/internal"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+	"github.com/spf13/cobra"
+)
+
+var statCMD = &cobra.Command{
+	Use:   "stat",
```

Comment:
`status` or `state`, `stat` usually stands for statistics

Reply:
i was inspired by linux: https://linux.die.net/man/2/stat, it says "get file status"

Reply:
but mb that is not clear enough, so let it be `status`